hub/constraints/instruction-handling/call/generalities/3rd_set_of_account_rows.lisp

```diff
@@ -18,7 +18,8 @@
 (defun    (call-instruction---summon-callee-account-thrice)    (*    PEEK_AT_SCENARIO
                                                                      scenario/CALL_SMC_FAILURE_CALLER_WILL_REVERT))
 
-(defconstraint    call-instruction---3rd-callee-account-operation                     (:guard (call-instruction---summon-both-account-rows-twice-or-more))
+(defconstraint    call-instruction---3rd-callee-account-operation
+                  (:guard (call-instruction---summon-callee-account-thrice))
```

Comment:
This was using the wrong guard, thus triggering those constraints in other cases besides `SMC_FAILURE_WILL_REVERT`.